template3b-images.js

```diff
@@ -2,35 +2,61 @@
 //  of a superclass (constructor Card)
 var ImageCard = (function(){
 
-	// Create subclass constructor:
-	function Ctor(id) {
-		// call superclass ctor first,
-		// then add subclass-specific properties
-	}
-	// Replace default prototype so that subclass inherits from superclass...
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-	return Ctor; //== ImageCard
+    // Create subclass constructor:
+    function Ctor(id) {
+        // call superclass ctor first,
+        Card.call(this, id);
+        // then add subclass-specific properties
+    
+    }
+    // Replace default prototype so that subclass inherits from superclass...
+    Ctor.prototype = Object.create(Card.prototype);
+    Ctor.prototype.constructor = Ctor;
+
+    Ctor.prototype.render = function() {
+      var myHtml = '<img class="cardImage" src="images/SVG-cards-1.3/' + this.imgName() + '.svg">';
+      document.body.innerHTML += myHtml;
+      console.log("Hello, I'm card number " + this.id + " and I'm the " + this.name());
```

Comment:
Line 19 is where we seem to be encountering issues. If we comment out this line your function is rendering cards to the DOM just fine. More specifically it is your 'this.name()' function that is throwing your error, due to its inability to access rankNames as a function. So, somehow this.name() needs to be able to privately access an array of rank names. Looking good otherwise!